internal/it/nearcache_util.go

```diff
@@ -139,9 +140,29 @@ func (tcx *NearCacheTestContext) AssertNearCacheContent(size int64, valueFmt str
 			}
 		}
 		target := fmt.Sprintf(valueFmt, i)
-		if !assert.Equal(tcx.T, target, value) {
-			tcx.T.FailNow()
-		}
-		// TODO: assertNearCacheRecord
+		require.Equal(tcx.T, target, value)
+		tcx.AssertNearCacheRecord(getRecordFromNearCache(nca, i), i, tcx.Config.InMemoryFormat)
 	}
 }
+
+func (tcx *NearCacheTestContext) AssertNearCacheRecord(rec *inearcache.Record, key int64, imf nearcache.InMemoryFormat) {
+	// see: com.hazelcast.internal.nearcache.impl.NearCacheTestUtils#assertNearCacheReference
```

Comment:
My guess is it should be as follows: 

```suggestion
	// see: com.hazelcast.internal.nearcache.impl.NearCacheTestUtils#assertNearCacheRecord
```

Reply:
Fixed at: https://github.com/hazelcast/hazelcast-go-client/pull/807/commits/0d49f98e13b8fc0d4843db1366d353fb6a96a174